src/paymentservice/README.md

```diff
@@ -40,6 +40,21 @@ The default health is available at `/paymentservice/actuator/health`
  
 See [Swagger-UI endpoint](http://localhost:8081/paymentservice/swagger-ui.html)
 
+### Splunk
+
+Payment services support [Splunk](https://www.splunk.com/) for event logging. To enable splunk run the mvn commend with the splunk profile activated
+
+```
+mvn package -P splunk
+```
+
```

Comment:
Just a comment, but should we be able to run "mvn spring-boot:run -P splunk" too? This would start the splunk profile, correct?

Reply:
yes that's right

Reply:
let me add this command to

Reply:
added